src/parser/typedefs.ts

```diff
@@ -0,0 +1,13 @@
+import { FabricObject } from '../shapes/Object/FabricObject';
+
+export type TSvgParsedCallback = (
+  results: FabricObject[] | null,
+  options?: Record<string, any>,
+  elements?: SVGElement[],
+  allElements?: SVGElement[]
```

Comment:
Yes this is wrong, not sure why i did it, those are passed in regardless, i think i did it because i wondered that when null was there i wouldn't have the other 3, but then i mocked them.

Reply:
This is also automatically solved in the moment we move those to promise, since those will reject.